systemtest/src/test/java/io/strimzi/systemtest/operators/NamespaceDeletionRecoveryST.java

```diff
@@ -208,11 +212,17 @@ void testTopicNotAvailable(ExtensionContext extensionContext) throws Interrupted
     private void prepareEnvironmentForRecovery(ExtensionContext extensionContext, String topicName, int messageCount) {
         String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
 
-        // Setup Test environment with Kafka and store some messages
-        prepareEnvForOperator(extensionContext, NAMESPACE);
-        applyBindings(extensionContext, NAMESPACE);
-        // 060-Deployment
-        resourceManager.createResource(extensionContext, BundleResource.clusterOperator(NAMESPACE).build());
+        new SetupClusterOperator.SetupClusterOperatorBuilder()
+            .withExtensionContext(extensionContext)
+            .withClusterOperatorName(Constants.STRIMZI_DEPLOYMENT_NAME)
+            .withNamespace(NAMESPACE)
+            .withWatchingNamespaces(NAMESPACE)
```

Comment:
I guess that when the `watching namespaces` is not set, it will take the value from `namespace` right? So you don't need it here?

Reply:
correct I will update it ))